editor/editor_file_system.h

```diff
@@ -310,6 +310,7 @@ class EditorFileSystem : public Node {
 	void scan();
 	void scan_changes();
 	void update_file(const String &p_file);
+	void update_files(const Vector<String> *p_script_paths);
```

Comment:
Any reason to pass by pointer? It's never modified, should probably be passed like so instead:
```suggestion
	void update_files(const Vector<String> &p_script_paths);
```

Reply:
You are right, this is a vestige of some other tests.